alma/alma-installments-prestashop
# Create default insurance data in BO on plugin installation

### Reason for change

[Linear task](https://linear.app/almapay/issue/MPP-964/create-default-insurance-data-in-bo-on-plugin-installation)

### Code changes

- Add Database insurance if not exist
- Add default product insurance if not exist
- Add default attribute insurance if not exist

### How to test

Install the module and check if the DB, Product Insurance and Attribute Insurance are installed correctly
And retry to install or reset the module and check there are not double product and attribute.

### Checklist for authors and reviewers




### Non applicable
